FAQ.md

```diff
@@ -58,112 +50,79 @@ types of programming languages.
-c. create a strong feedback loop to improve Slint.
-
-#### When does the Ambassador license run out?
-
-The license grant is [perpetual](#what-does-perpetual-mean), which means that you can use Slint free-of-charge forever. The perpetual nature of the license also implies that even if we need to modify the terms of the license in the future, the modified terms will not apply to already granted licenses.
-
-#### For how long do you plan to offer the free-of-charge Ambasssador license?
-
-Forever. However, we may modify the terms of the license in the future based on user feedback and business needs.
+Refer to GPL FAQ [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL).
 
-#### How can I get the Ambassador license?
+#### My MIT-licensed program links to Slint GPLv3. Can someone fork my program to build and distribute a proprietary program?
 
-The license is automatically granted on signing the license agreement [here](https://slint-ui.com/ambassador-program.html#application). The authorized signatory could be the code owner, primary maintainer, or in case of an organisation, the relevant person authorized to sign contracts on behalf of the organisation.
+Yes, provided the person distributing the proprietary program acquired a Slint proprietary license, such as the Slint Royalty-free license or a paid license, instead of using Slint under GPLv3. The other option would be to remove the dependency to Slint altogether.
 
-#### Do all contributors to my code have to sign up for the Ambassador license?
+#### My MIT-licensed program links to Slint GPLv3. How can I convey to someone that they can distribute my program as part of a proprietary licensed program?
 
-No. All contributions to the code is covered under the Ambassador license. Hence, contributors can use Slint under the same license within this scope.
+You can add a note as part of your license that to distribute a proprietary licensed program, one can acquire a Slint proprietary license or the dependency to Slint should be removed.
 
-### Distributions
+#### My MIT-licensed program links to Slint GPLv3. Under what license can I release the binary of my program?
 
-#### Do I need to pay to distribute my application on desktop?
+While your software modules can remain under the MIT-license, the work as a whole must be licensed under the GPL.
+Hence, the binary must be made available under the GPLv3.
 
-No. Our proprietary license includes unlimited distribution on desktop.
+#### Scenario: Alice is a software developer, she wants her code to be licensed under MIT. She is developing an application "AliceApp" that links to Slint GPLv3. Alice also wants to allow that Bob, a user of AliceApp, can fork AliceApp into a proprietary application called BobApp
 
-#### Do I need to pay to distribute my application on embedded devices?
+##### Can Alice use the MIT license header to the source code of AliceApp application?
```

Comment:
Side-note: These very deep markdown nesting don't really render very effectively anymore - so it all becomes a text with the same font size.

I suggest to reformatting it as either a table or (preferred) a bullet list.